tests/zfs-tests/include/libtest.shlib

```diff
@@ -3158,10 +3158,11 @@ function zed_stop
 	if [[ -f ${ZEDLET_DIR}/zed.pid ]]; then
 		zedpid=$(cat ${ZEDLET_DIR}/zed.pid)
 		kill $zedpid
-		wait $zedpid
+		while ps -p $zedpid > /dev/null; do
+			sleep 1
+		done
```

Comment:
`wait` (shell-builtin) doesn't seem to work when `zed_start()` and `zed_stop()` are run in different processes (setup.ksh and cleanup.ksh respectively): this resulted in several failures running `fault` tests with `TEST_ZFSTESTS_ITERS="10"`.